logger_block.py

```diff
@@ -22,6 +22,8 @@ class Logger(TerminatorBlock):
     log_at = SelectProperty(LogLevel, title="Log At", default="INFO")
     log_as_list = BoolProperty(title="Log as a list",
                                default=False, visible=False)
+    log_hidden_attributes = BoolProperty(title="Log Hidden Attributes",
+                                         default=False)
```

Comment:
@hansmosh should this be visible by default?

Reply:
I'm ok with keeping it visible for now. If it confuses people, we can make it hidden later.